data/actionsmap/yunohost.yml

```diff
@@ -1289,6 +1289,36 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
```

Comment:
Automatically when the package is upgraded by apt, we can't ask the user to do that everytime and that would increase the cost of having a new migration a lot which would defeat the purpose of being able to reduce the cost of modification for us.